### PR TITLE
Assignment 4

### PR DESCRIPTION
**Pull request by davidhernandog**

### CODE REVIEW

File: Assignment4/davidhernandog_D21C027/task06.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+g.add((ns.JaneSmith, vcard.FullName, Literal('Jane Smith')))
+g.add((ns.JaneSmith, vcard.GivenName, Literal("Jane")))
+g.add((ns.JaneSmith, vcard.FamilyName, Literal('Smith')))
+for s, p, o in g:
+  print(s,p,o)
+
+"""**TASK 6.5: Add UPM as the university where John Smith works**"""
+
+g.add((ns.UPM, RDF.type, ns.University))
+g.add((ns.UPM, vcard.member, ns.JohnSmith))
```
Comment (dgarijo) [inline Assignment4/davidhernandog_D21C027/task06.py]: vcard does not define `member`. Find another vocabulary, or define your own property

File: Assignment4/davidhernandog_D21C027/task07.py [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+from rdflib.plugins.sparql import prepareQuery
+ns = Namespace("http://somewhere#")
+vcard = Namespace("http://www.w3.org/2001/vcard-rdf/3.0#")
+
+#OPTION1: RDFLib
+for s,p,o in g.triples((None, RDFS.subClassOf, ns.Person)):
+  print(s)
+for x,y,z in g.triples((None, RDFS.subClassOf, s)): 
+  print(x)
```
Comment (dgarijo) [inline Assignment4/davidhernandog_D21C027/task07.py]: You are listing all subclasses of person and then only the subclasses of the first subclass

**Review by dgarijo — CHANGES_REQUESTED** [review]
small changes required still

**Comment by davidhernandog** [comment]
I just changed it

**Review by dgarijo — APPROVED** [review]

**Pull request merged**